apps/financial-aid/backend/src/app/modules/staff/staff.service.ts

```diff
@@ -73,39 +77,74 @@ export class StaffService {
     input: CreateStaffDto,
     municipalityName: string,
     user: Staff,
+    isFirstStaffForMunicipality: boolean,
   ) {
-    if (input.roles.includes(StaffRole.EMPLOYEE)) {
-      try {
+    const contact = {
+      from: {
+        name: user.name,
+        address: user.email,
+      },
+      replyTo: {
+        name: user.name,
+        address: user.email,
+      },
+      to: input.email,
+    }
+
+    try {
+      if (
+        input.roles.includes(StaffRole.EMPLOYEE) &&
+        input.roles.includes(StaffRole.ADMIN)
+      ) {
+        await this.emailService.sendEmail({
+          ...contact,
+          subject: 'Vinnsluaðili og stjórnandi í vinnslukerfi fjárhagsaðstoðar',
+          html: AdminAndEmployeeEmailTemplate(
+            municipalityName,
+            environment.veitaUrl,
+            input.email,
+            isFirstStaffForMunicipality,
+          ),
+        })
+      } else if (input.roles.includes(StaffRole.EMPLOYEE)) {
         await this.emailService.sendEmail({
-          from: {
-            name: user.name,
-            address: user.email,
-          },
-          replyTo: {
-            name: user.name,
-            address: user.email,
-          },
-          to: input.email,
-          subject: 'Aðgangur fyrir vinnslukerfi fjárhagsaðstoðar veittur',
+          ...contact,
+          subject: 'Vinnsluaðili í vinnslukerfi fjárhagsaðstoðar',
           html: EmployeeEmailTemplate(
             municipalityName,
             environment.veitaUrl,
             input.email,
           ),
         })
-      } catch (error) {
-        logger.warn('failed to send email', error)
+      } else if (input.roles.includes(StaffRole.ADMIN)) {
```

Comment:
Maybe have a switch statement here depending on staffrole ? Just suggestion 🤷‍♀️  Feel free to ignore